db/system_keyspace.cc

```diff
@@ -73,25 +73,44 @@ std::unique_ptr<query_context> qctx = {};
 static logging::logger slogger("system_keyspace");
 static const api::timestamp_type creation_timestamp = api::new_timestamp();
 
-bool system_keyspace::is_extra_durable(const sstring& name) {
-    return boost::algorithm::any_of(extra_durable_tables, [name] (const char* table) { return name == table; });
-}
-
-static const auto set_null_sharder = schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
-    // tables in the "system" keyspace which need to use null sharder
-    static const std::unordered_set<sstring>& system_ks_null_shard_tables = {
-        db::schema_tables::SCYLLA_TABLE_SCHEMA_HISTORY,
-        db::system_keyspace::RAFT,
-        db::system_keyspace::RAFT_SNAPSHOTS,
-        db::system_keyspace::RAFT_SNAPSHOT_CONFIG,
-        db::system_keyspace::GROUP0_HISTORY,
-        db::system_keyspace::DISCOVERY,
-        db::system_keyspace::BROADCAST_KV_STORE,
-    };
-    if (ks_name == db::system_keyspace::NAME && system_ks_null_shard_tables.contains(cf_name)) {
-        props.use_null_sharder = true;
-    }
-});
+namespace {
+    const auto set_null_sharder = schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
+        // tables in the "system" keyspace which need to use null sharder
+        static const std::unordered_set<sstring>& system_ks_null_shard_tables = {
+            schema_tables::SCYLLA_TABLE_SCHEMA_HISTORY,
+            system_keyspace::RAFT,
+            system_keyspace::RAFT_SNAPSHOTS,
+            system_keyspace::RAFT_SNAPSHOT_CONFIG,
+            system_keyspace::GROUP0_HISTORY,
+            system_keyspace::DISCOVERY,
+            system_keyspace::BROADCAST_KV_STORE,
+        };
+        if (ks_name == db::system_keyspace::NAME && system_ks_null_shard_tables.contains(cf_name)) {
+            props.use_null_sharder = true;
+        }
+    });
+
+    const auto set_wait_for_sync_to_commitlog = schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
+        static constexpr const char* extra_durable_tables[] = {
```

Comment:
Why not unordered_set<> like in others of the like?

Reply:
Just moved the code "as is". Ok, will make it uniform.